tests/unit/spec/base/core-datalayer.js

```diff
@@ -28,23 +27,24 @@ describe('core-datalayer.js', function() {
 
     describe('pageHasVideo', function() {
 
-        afterEach(function() {
-            $('#htmlPlayer').remove();
-            $('#iframePlayer').remove();
-        });
-
```

Comment:
Should both of the `.remove()` calls be replaced?

Reply:
Same as above, the two tests don't share any common cleanup code, so I moved it to each respective test.

Reply:
Same as above, the third test doesn't modify the DOM, so I moved the code to the relevant tests where cleanup is needed.